lib/treelify/acts_as_user.rb

```diff
@@ -32,6 +32,15 @@ def member_for(resource)
         resource.hierarchy.members.where("members.user_id": id).first
       end
 
+      def revoke(resource)
```

Comment:
I can't revoke specific role only? (all as default is cool, but would be nice to have a chance to revoke specific - in that way, I can use this gem with inheritance or not - so I can use it instead of Rolify for example, but still have a chance to switch anytime to "tree" mode).